esy-lib/Checksum.re

```diff
@@ -51,6 +51,19 @@ let of_yojson = json =>
   | _ => Error("expected string")
   };
 
+let convOpamKind = kind =>
```

Comment:
We had to add `opam-state` and other opam deps here - it would be nice to avoid introducing them here. Perhaps, `convOpamKind` and `ofOpamHash` could reside inside `esy-install` that already needs opam libraries?

Reply:
Makes sense